editor/components/TableMap.tsx

```diff
@@ -71,22 +74,57 @@ export const TableMap = ({ mapRef, geojson, mapConfig }: TableMapProps) => {
     }
   }, [geojsonBounds, mapRef]);
 
+  // todo, check if this is the right type to use
```

Comment:
does this TODO need to stick around? it looks like the right type when i inspected in VSCode 👍